### ISSUE TITLE
use github annotations to promote errors

### ISSUE THREAD

**Issue by staabm** [gr2m/twitter-together#146]
**What’s missing?**

atm I need to lookup the gihtub action logs to find out about errors of the action.

**Why?**

it would be awesome this action would just use github annotations - see a live example at https://github.com/staabm/gh-annotation-example/pull/1/files - so the actual errors get shown in the Pull Requests

its a pretty simple format you need to use, so github actions will automatically turn errors/infos/warnings into annotations.

example:

https://github.com/staabm/gh-annotation-example/blob/0f34e65c629524aae166a617f97a7800a45194db/.github/workflows/annotate.yml#L14-L18

**Alternatives you tried**

none.

**Comment by gr2m** [comment]
good idea! I saw your other PRs, just very busy right now :( I have them on my backlog, I'll get to them eventually
